src/components/ActivityIndicator/index.tsx

```diff
@@ -62,7 +63,7 @@ const ActivityIndicator: FunctionComponent<ActivityIndicatorProps> = ({
   size = 40,
   strokeWidth = 16,
   color = 'primary',
-  trailColor = 'gray350',
+  trailColor = 'primary',
```

Comment:
Should be neutral in order to keep same old behavior :) 

```suggestion
  trailColor = 'neutral',
```